### PR TITLE
[#174468075] New Community page

### PR DESCRIPTION
**Pull request by DmitryBarskov**
https://www.pivotaltracker.com/story/show/174468075

https://projects.invisionapp.com/d/main?origin=v7#/console/20307299/429460947/preview?scrollOffset=3315

### CODE REVIEW

**Comment by DmitryBarskov** [comment]
@IlgizGilmanov can you help me with the styles?

File: assets/javascripts/controllers/community-ui-builder.js.es6 [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          topic_template: "",
+          required_tag_group_name: "",
+          topic_featured_link_allowed: true,
+          search_priority: 0,
+          custom_fields: {
+            introduction_raw: this.introduction + "\n",
+            tenets_raw: this.tenets,
+          },
+        },
+      }).then(({ category }) => {
```
Comment (ShaidullinArtur) [inline assets/javascripts/controllers/community-ui-builder.js.es6:60]: We should show error message to the user if we will not be able to create category for some reason. You can use `popupAjaxError` like in `/communitarian/assets/javascripts/controllers/resolution-ui-builder.js.es6`

Снимок экрана 2020-09-01 в 15 31 31

**Review by ShaidullinArtur — COMMENTED** [review]
1 comment otherwise looks 🆗

**Review by Vadim-Kurnatovskiy — COMMENTED** [review]
Looks good but we need to add validation errors if they present

File: assets/javascripts/discourse/templates/modal/community-ui-builder.hbs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<div class="resolution-form">
+  {{#d-modal-body title=formTitle class="poll-ui-builder community-ui-builder"}}
+    <form class="poll-ui-builder-form form-horizontal" {{action "submitCommunity" on="submit"}}>
+      <section class="input-group poll-textarea">
+        <label>{{i18n "communitarian.community.ui_builder.title.label"}}</label>
+        {{text-field value=title
+          class="full-width"
+          tabindex="1"
```
Comment (IlgizGilmanov) [inline assets/javascripts/discourse/templates/modal/community-ui-builder.hbs]: Please tell me if there was any special reason to put down the `tabindex` attribute? If only for the order of switching between interactive elements, then by default (if you do not set this attribute) the order will be the same.

File: assets/javascripts/controllers/community-ui-builder.js.es6 (hunk 1 above)
Comment (IlgizGilmanov) [reply assets/javascripts/controllers/community-ui-builder.js.es6:60]: In addition, you can set the `disabled` attribute if the fields are empty.

**Review by Vadim-Kurnatovskiy — APPROVED** [review]

**Review by IlgizGilmanov — APPROVED** [review]
:+1:

**Review by ShaidullinArtur — APPROVED** [review]

**Pull request merged**